is/object_of_test.ts

```diff
@@ -98,4 +98,46 @@ Deno.test("isObjectOf<T>", async (t) => {
       >(true);
     }
   });
+
+  await t.step("if 'predObj' has prototype properties", async (t) => {
+    const prototypeObj = {
+      a: is.Number,
+      b: is.Boolean,
+    };
+    // deno-lint-ignore ban-types
+    const predObj2 = Object.assign(Object.create(prototypeObj) as {}, {
```

Comment:
I prefer to overwrite `predObj` here while this whole step only requires `predObj2` and we don't refer `predObj` of it's parent. (I assumed that overwriting constants in a child scope is accepted in TS, right?)